CONTRIBUTING.md

```diff
@@ -190,5 +191,8 @@ Please run your code through:
 
 7. [Open a Pull Request](https://help.github.com/articles/using-pull-requests/)
     with a clear title and description against the `master` branch. All tests must be passing before we will review the PR.
+    
+## Code Reviews<a name="code-reviews"></a>
```

Comment:
you do not need to insert an HTML anchor tag, github makes anchors automatically for every heading you make, inserting dashes in lieu of spaces

Reply:
the link would look like
```
[Code Review](#code-reviews)
```